bindings/core/src/method_handler/utils.rs

```diff
@@ -57,21 +57,30 @@ pub(crate) fn call_utils_method_internal(method: UtilsMethod) -> Result<Response
         UtilsMethod::HashTransactionEssence { essence } => Response::TransactionEssenceHash(prefix_hex::encode(
             TransactionEssence::try_from_dto_unverified(essence)?.hash(),
         )),
-        UtilsMethod::VerifyEd25519Signature {
-            signature,
-            message,
-            address,
-        } => {
-            let signature = Ed25519Signature::try_from(signature)?;
-            let msg: Vec<u8> = prefix_hex::decode(message)?;
-            let address = Ed25519Address::try_from(address)?;
-            Response::Bool(signature.is_valid(&msg, &address).is_ok())
-        }
         UtilsMethod::VerifyMnemonic { mut mnemonic } => {
             verify_mnemonic(&mnemonic)?;
             mnemonic.zeroize();
             Response::Ok
         }
+        UtilsMethod::VerifyEd25519Signature { signature, message } => {
+            let signature = Ed25519Signature::try_from(signature)?;
+            let message: Vec<u8> = prefix_hex::decode(message)?;
```

Comment:
down below we map the error to an `InvalidField` error, but here ... we don't :thinking: Is that intended?

Reply:
Idk, it's Alex's code. But I changed it.